src/DynamoPackages/PackageManagerClient.cs

```diff
@@ -16,10 +16,10 @@ public class PackageManagerClient
         // These were used early on in order to identify packages with binaries and python scripts
         // This is now a bona fide field in the DB so they are obsolete. 
 
-        [Obsolete] internal static readonly string PackageContainsBinariesConstant =
+        [Obsolete] public static readonly string PackageContainsBinariesConstant =
```

Comment:
It is being used to determine if old packages contains any binary or python script. It is used on PackageManagerSearchViewModel.PackageOnExecuted method. Followed the same.